README.md

```diff
@@ -214,6 +214,16 @@ in the `context["params"]` variable, e.g. getting a training data you would use
 
     Returns a dict with the feature drift data.
 
+- `UploadDatasetOperator`
```

Comment:
I would place it as the first operator in the list, as it would fit the logical flow of actions in DR pipeline.